Water-Physics/Assets/Code/Scripts/Environment/WaterFloat.cs

```diff
@@ -0,0 +1,133 @@
+﻿using UnityEngine;
+
+namespace Code.Scripts.Environment
+{
+    [RequireComponent(typeof(Rigidbody))]
+    public class WaterFloat : MonoBehaviour
+    {
+        //public properties
+        public float airDrag = 1;
+        public float waterDrag = 10;
+        public bool affectDirection = true;
+        public bool attachToSurface;
+        public Transform[] floatPoints;
```

Comment:
Volgens mij worden deze public variables niet door andere klassen gebruikt maar alleen door de inspector. Gebruik dan `[SerializedField] private` zoals beshreven in de style guide.